### ISSUE TITLE
scatter "angle" works with a constant but not with a column

### ISSUE THREAD

**Issue by brettbrocato** [PatrikHlobil/Pandas-Bokeh#4]
Thank you for your effort, Patrik!  I have a challenge with pointing the marker at an angle, but I'm not certain bokeh can do this generally.

My dataframe has ~3400 rows and 3 columns.  
df

> Out[xxx]: 
>            Xnum   Ynum  col3
> ..................

I have already eliminated NaNs: 

`df=df.dropna(axis=0,how='any')`

I have tried this, it produces a graph on /Interactive%20Plot.html, and I can successfully change the angle:

`df.plot_bokeh(kind="scatter",x="Xnum",y="Ynum" ,marker="triangle", size=8,angle=0.5,line_width=0.001,figsize=(400, 400) )`

When I run this: 
`df.plot_bokeh(kind="scatter",x="Xnum",y="Ynum" ,marker="triangle", size="col3",angle=0.5,line_width=0.001,figsize=(400, 400) )`

I get the error and no plot:

> E-1001 (BAD_COLUMN_NAME): Glyph refers to nonexistent column name: col3 [renderer: GlyphRenderer(id='617da22f-e32c-4004-b8c6-a13be540d385', ...)]

**Comment by PatrikHlobil** [comment]
Hi, thanks a lot for your feedback. Sorry for the late reply, but I haven't looked at the issues for a while because I was the only person working on the library so far. If you find more problems/issues, please report them and I try to respond faster in the future ;D.

The problem should be fixed with the update 0.1. I hope to release it this week. The problem is that I in the current release 0.0.2 only the columns specified in x and y parameters are embedded into the plots and therefore only these are accessible with the additional keyword arguments. An example can be found in the updated [README](https://github.com/PatrikHlobil/Pandas-Bokeh/blob/master/README_new.md):

```python
#Load Iris Dataset from Scikit Learn:
from sklearn.datasets import load_iris
iris = load_iris()
df = pd.DataFrame(iris["data"])
df.columns = iris["feature_names"]
df["species"] = iris["target"]
df["species"] = df["species"].map(dict(zip(range(3), iris["target_names"])))
df = df.sample(frac=1)

p_scatter = df.plot_bokeh.scatter(
    x="petal length (cm)",
    y="sepal width (cm)",
    category="species",
    title="Iris DataSet Visualization",
    size="sepal length (cm)")
```
![scatterplot2](https://user-images.githubusercontent.com/31643597/50249153-82e1c280-03dd-11e9-9216-dce2e7180f30.png)

You can barely see the differences in size in the example, but it works now.

If you do not want to wait, you can install it from the following wheel:

[pandas_bokeh-0.1-py2.py3-none-any.zip](https://github.com/PatrikHlobil/Pandas-Bokeh/files/2696563/pandas_bokeh-0.1-py2.py3-none-any.zip)

You just have to:

* unzip the file
* pip uninstall pandas-bokeh
* pip install <path to pandas_bokeh-0.1-py2.py3-none-any.whl>

If it isn't urgent, just wait for the 0.1 release.

Best Patrik
